Applications/Spire/Include/Spire/Ui/ScrollBar.hpp

```diff
@@ -0,0 +1,94 @@
+#ifndef SPIRE_SCROLL_BAR_HPP
+#define SPIRE_SCROLL_BAR_HPP
+#include <QScrollBar>
+#include "Spire/Styles/StyledWidget.hpp"
+#include "Spire/Ui/Ui.hpp"
+
+namespace Spire {
+namespace Styles {
+
+  /** Sets the width. */
+  using Width = BasicProperty<int, struct WidthTag>;
```

Comment:
Is it possible to avoid duplicating this? It seems like these size properties would be common to all StyledWidgets.

Reply:
Yes, they should be in a place where all common properties are declared. What is a better place? I find lots of properties are in Box.hpp, so do I put them to Box.hpp?